### ISSUE TITLE
Unable to download from baidu

### ISSUE THREAD

**Issue by Horopter** [yao8839836/ProbaseLDA#1]
I tried accessing the link on https://pan.baidu.com/s/1qXvIgXI but the server keeps on giving 403 error. The only foreign language I know in English and I seek your help in downloading the isa_core.sql document.

**Comment by yao8839836** [comment]
@Horopter , Hi, can you open this link？
https://pan.baidu.com/s/1qXvIgXI#list/path=%2Fdataset%2Fprobase_sql&parentPath=%2Fdataset

![image](https://user-images.githubusercontent.com/7744044/36635448-77eb48f4-197a-11e8-9ef0-3273f0cd9a9a.png)

In this figure "下载" means download. you can check the boxes to choose files.

Thanks!

**Comment by Horopter** [comment]
No, my issue is that the server is very slow in terms of download speed. Gives less than 72KBPS. And then the server gives 403 error. If you could please upload it to a dropbox or google drive, that'd be great. Baidu services are very slow in south Asian provinces.

**Comment by yao8839836** [comment]
@Horopter Hi, I uploaded the files to Google drive:
https://drive.google.com/open?id=1zgbkhOAbCojyRxC_eoP9DeEaORUUO2B7

**Comment by Horopter** [comment]
Oh, it's an actual sql file. Is there a csv version of these files?

**Comment by yao8839836** [comment]
@ No, I downloaded tsv files from Microsoft Probase project page in 2014 and transformed them to sql files. You may want to download the original file from the project page: http://probase.msra.cn/dataset.aspx. The page has been updated. You may contact the team members of the Microsoft Probase project.
